### DISCUSSION TITLE
design concept

### DISCUSSION THREAD

**Discussion by FunctionalMetatable** [FlagClicked/FlagClicked#78] [Show and tell]
logged out homepage design concept
![fc](https://user-images.githubusercontent.com/67440879/120013094-3e0fdc00-bfd8-11eb-8716-cad018bd9937.png)

**Comment by danielgallab** [comment]
Epic. But the tutorial on the right needs some discussion I think.

**Comment by markverb1** [comment]
Looks epic. Almost like scratch.

**Comment by CubeyTheCube** [comment]
I don't like the font. And is "Start Learning" intentionally on the right side of the button?

**Reply by FunctionalMetatable** [reply]
yeah. i was gonna put a little icon there but i forgot

**Comment by FunctionalMetatable** [comment]
Also, the video at #11 could be replacing the background of the main banner

**Comment by Steve0Greatness** [comment]
hmm, looks pretty good.
I think it would be ok.
